cypress/integration/smoke/balances.spec.js

```diff
@@ -0,0 +1,104 @@
+const assetsTableContainer = '.MuiTableContainer-root'
+const balanceRowTestId = '[data-testid=balance-row]'
+const receiveModalClass = '.receive-modal'
+
+const SAFE_TEST = '0x8675B754342754A30A2AeF474D114d8460bca19b'
+const ASSETS_LENGTH = 26
+
+describe('Assets > Coins', () => {
+  const number = 123456.789
+  const localeNumber = number.toLocaleString()
+  // Find localeNumber thousand separator and decimal separator
+  const [thousandSeparator, decimalSeparator] = localeNumber.match(/[^0-9]/g)
+
+  // Fiat balance regex
+  const fiatRegex = new RegExp(`([0-9]{1,3}${thousandSeparator})*[0-9]{1,3}${decimalSeparator}[0-9]{2}`)
+  // Token balance regex
+  const tokenRegex = new RegExp(`([0-9]{1,3}${thousandSeparator})*[0-9]{1,3}${decimalSeparator}[0-9]{5}`)
+
+  beforeEach(() => {
+    // Open the Safe used for testing
+    cy.visit(`/eth:${SAFE_TEST}/balances`)
+  })
+
+  describe('Should have all table entries', () => {
+    it('Should have 26 entries in the table', () => {
```

Comment:
Please start the string with a lowercase letter. It should read like a single sentence that starts with `it`. I.e. `it should have 26 entries`.